omod/src/main/java/org/openmrs/module/kenyaemr/web/controller/KenyaemrCoreRestController.java

```diff
@@ -410,6 +411,49 @@ public Object getDefaultConfiguredFacility() {
 
 	}
 
+    /**
+	 * ARV drugs
+	 *
+	 * @return custom ARV drugs object for non standard regimen
+	 */
+	@RequestMapping(method = RequestMethod.GET, value = "/arvDrugs")
+	@ResponseBody
+	public Object getArvDrugs() {
+        ConceptService concService = Context.getConceptService();
+        ArrayNode drugs = JsonNodeFactory.instance.arrayNode();
+        ObjectNode drugsObj = JsonNodeFactory.instance.objectNode();
+
+		List<Concept> arvDrugs = Arrays.asList(
+				concService.getConcept(84309),
+				concService.getConcept(86663),
+				concService.getConcept(84795),
+				concService.getConcept(78643),
+				concService.getConcept(70057),
+				concService.getConcept(75628),
+				concService.getConcept(74807),
+				concService.getConcept(80586),
+				concService.getConcept(75523),
+				concService.getConcept(79040),
+				concService.getConcept(83412),
+				concService.getConcept(71648),
+				concService.getConcept(159810),
+				concService.getConcept(154378),
+				concService.getConcept(74258),
+				concService.getConcept(164967)
+		);
+
+		for (Concept con: arvDrugs) {
+            ObjectNode node = JsonNodeFactory.instance.objectNode();
+            node.put("name", con.getName() != null ? con.getName().toString() : "");
+            node.put("name", con.getUuid() != null ? con.getUuid().toString() : "");
```

Comment:
I think this line will overwrite the previous one. Did you mean to do
node.put("uuid"....) or something?

Reply:
Sure! updated. Thanks for the catch